content/guides/concepts/cid.md

```diff
@@ -5,18 +5,39 @@ menu:
         parent: concepts
 ---
 
-A *content identifier* is a value that addresses a single piece of content in IPFS. It is mainly a cryptographic hash of the content, but is encoded as a [multihash](https://github.com/multiformats/multihash) and [multicodec](https://github.com/multiformats/multicodec). (Note: older CIDs have a different design — see [version 0](#version-0) below.)
+## Summary
 
-<!-- TODO: explain more of the details of how CID v1 is composed here. -->
+A *content identifier*, or CID, is a label used for addressing content in IPFS. CID's are used as a standard way of pointing to pieces of information.  CID's identify specific pieces of content stored in IPFS.
 
-You can read up on the details in the [CID spec](https://github.com/ipld/cid). You might also want to check out the [CID inspector](http://cid-utils.ipfs.team/#zb2rhiVd5G2DSpnbYtty8NhYHeDvNkPxjSqA7YbDPuhdihj9L) for an interactive breakdown of CIDs.
+CID's are based on that content's cryptographic hash - a different piece of content will have a different hash and will produce different CID's.
 
-## Version 1
+## Format of CID's
+
+CID's can take a few different forms, each easy for humans and/or software to decode.  Any specific CID can be transformed to other equivalent CID representations (for example, using different base, CID version or codec).  
+
+CID v1 and later are comprised of some leading identifiers making it easy to identify which representation is used, along with the content-hash itself. In v1 and later, these include a multibase identifier, [multicodec](https://github.com/multiformats/multicodec) identifier, and CID version-identifier:
```

Comment:
We should not say “and later” (there is no v2 and we have no idea what it might look like if and when it’s invented).

This paragraph and most of the stuff down to the next heading are really particular to the v1 format and doesn’t apply to v0 at all. I think you should probably just move it all under the “version 1” heading. I’m not sure if we actually need a “format” section alongside a “versions” section, since each version is actually a different format.

I think the only thing we really need to say at this point (before the version sections) is that there are two versions, and IPFS is slowly migrating to v1 by default. (Maybe we could add which commands do v0 by default and which do v1 by default).

Reply:
According to the multiformat strategy, v2 (if/when) will at least have a compatible leading identifier to indicate that it is v2.  But okay, I'll hold back on that aspect.  Thanks.